pronto/api/annotation.py

```diff
@@ -407,31 +417,87 @@ def create_annotation():
             )
             ann_id, = cur.fetchone()
 
-            return jsonify({
-                "status": False,
-                "error": {
-                    "title": "Database error",
-                    "message": f"Duplicate of {ann_id}"
-                }
-            }), 500
+            return (
+                None,
+                {
+                    "status": False,
+                    "error": {
+                        "title": "Database error",
+                        "message": f"Duplicate of {ann_id}"
+                    }
+                },
+                500,
+            )
         else:
-            return jsonify({
-                "status": False,
-                "error": {
-                    "title": "Database error",
-                    "message": f"The annotation could not be created: {exc}."
-                }
-            }), 500
+            return (
+                None,
+                {
+                    "status": False,
+                    "error": {
+                        "title": "Database error",
+                        "message": f"The annotation could not be created: {exc}."
+                    }
+                },
+                500,
+            )
     else:
         con.commit()
-        return jsonify({
-            "status": True,
-            # RETURNING -> getvalue() returns an array
-            "id": ann_id.getvalue()[0]
-        }), 200
+        return (
+            ann_id,
+            {
+                "status": True,
+                # RETURNING -> getvalue() returns an array
+                "id": ann_id.getvalue()[0]
```

Comment:
Make sure the first item of the returned tuple is a `str`, not an `oracledb.Var` object:

```suggestion
        # RETURNING -> getvalue() returns an array
        ann_id = ann_id.getvalue()[0]
        return (
            ann_id,
            {
                "status": True,
                "id": ann_id
```